react/features/poll/components/Poll/web/CreatePollButton.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+
+import { openPollCreationPage } from '../../../actions';
+import {
+    PollCreateButton
+} from '../../../styled';
+
+/**
+ * CreatePollButton component.
+ *
+ * @returns {any} Arg.
+ */
+export function CreatePollButton() {
+    const dispatch = useDispatch();
+    const { t } = useTranslation();
```

Comment:
`t` isn't being used, we need to ensure our code has passed the linter rules. https://github.com/jitsi/jitsi-meet/blob/master/CONTRIBUTING.md#creating-pull-requests